test/BondingERC20TokenFactory.t.sol

```diff
@@ -72,6 +74,25 @@ contract BondingERC20TokenFactoryTest is Test {
     );
   }
 
+  function testCreationFee() public {
+    vm.prank(owner);
+    factory.updateCreationFee(0.01 ether);
+
+    vm.expectRevert();
+    factory.deployBondingERC20Token{value: 0.001 ether}("ERC20Token", "ERC20");
+
+    factory.deployBondingERC20Token{value: 0.01 ether}("ERC20Token", "ERC20");
```

Comment:
All good, I see that as a owner, we are updating the creation fee with the `prank` cheatcode and it only applies for the next function call which ends at line 79. Now at line 84, a token is deployed by the test smart contract, can you change it to a user calling `factory.deployBondingERC20Token{value: 0.01 ether}("ERC20Token", "ERC20");` ?

Reply:
yeah. can do that. but the goal here was to test the creation fee. So, deployer doesn't matter. the thing that matters is that treasury should get the fees once the token is deployed regardless of deployer

Reply:
decided to not do this to keep similarity with other tests in this file. For All the other tests, the deployer is test contract.